src/Bridge/Doctrine/EventListener/PublishMercureUpdatesListener.php

```diff
@@ -88,6 +89,17 @@ public function __construct(ResourceClassResolverInterface $resourceClassResolve
         $this->graphQlSubscriptionManager = $graphQlSubscriptionManager;
         $this->graphQlMercureSubscriptionIriGenerator = $graphQlMercureSubscriptionIriGenerator;
         $this->reset();
+
+        $rawurlencode = ExpressionFunction::fromPhp('rawurlencode', 'escape');
+        $this->expressionLanguage->addFunction($rawurlencode);
```

Comment:
Also with a real request I got: `Registering functions after calling evaluate(), compile() or parse() is not supported.`

Reply:
What would an actual request be? Can you provide an example?

Reply:
This one is weird. I don't know where it should be put if not in the constructor. Maybe we need to use an expression provider (https://symfony.com/doc/current/components/expression_language/extending.html#using-expression-providers)?

Reply:
I believe that the ```Expression Provider``` won't solve in the same way, but since I don't know why, I asked for the example to try to reproduce it, because when I did the addition it worked normally

Reply:
> What would a real request? Can you provide an example?

Broswer request to the api :) will check if its reproduceable everytime.

Reply:
IMHO as @alanpoulain said it would be better to use expression provider as it is done for security part https://github.com/api-platform/core/blob/main/src/Security/Core/Authorization/ExpressionLanguageProvider.php